common/kvstore/pom.xml

```diff
@@ -89,7 +89,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-slf4j-impl</artifactId>
+      <artifactId>log4j-slf4j2-impl</artifactId>
```

Comment:
https://logging.apache.org/log4j/2.x/log4j-slf4j-impl/index.html

It seems that `log4j-slf4j-impl` and `log4j-slf4j2-impl` should not be used together